backend/src/routes/api/accelerators/acceleratorUtils.ts

```diff
@@ -63,7 +63,9 @@ export const getAcceleratorNumbers = async (
     )
     .catch((e) => {
       fastify.log.error(
-        `Exception when listing cluster nodes: ${e.response?.body?.message || e.message || e}`,
+        `A ${e.statusCode} error occurred when listing cluster nodes: ${
+          e.response?.body?.message ?? e.statusMessage
```

Comment:
Not usre why don't we use the nullish coalescing operator over or, having falsy values can be a pain in the ass sometimes, but curious why you decide to use it here. (it's just a nit)

Reply:
used it because other implementations used it. I can fix it